ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendProvider.scala

```diff
@@ -98,7 +98,9 @@ case class TestBackend(
     participantPartyStorageBackend: PartyRecordStorageBackend,
     metering: TestMeteringBackend,
     identityProviderStorageBackend: IdentityProviderStorageBackend,
-)
+) {
+  val pruningDtoQeuries: PruningDtoQueries = new PruningDtoQueries
```

Comment:
Let's not single him out and add him as constructor param

Reply:
Ok